vpr/src/timing/timing_graph_builder.cpp

```diff
@@ -50,13 +243,16 @@ std::unique_ptr<TimingGraph> TimingGraphBuilder::timing_graph(bool allow_danglin
     return std::move(tg_);
 }
 
+//Constructs the timing graph from the netlist
 void TimingGraphBuilder::build(bool allow_dangling_combinational_nodes) {
     tg_ = std::make_unique<tatum::TimingGraph>();
 
     // Optionally allow dangling combinational nodes.
     // Set by `--allow_dangling_combinational_nodes on`. Default value is false
     tg_->set_allow_dangling_combinational_nodes(allow_dangling_combinational_nodes);
 
+    //Walk through the netlist blocks adding the associated nodes and internal edges
```

Comment:
Maybe this comment is better as a little more explicit.  If I follow the top-level comments, this look is constructing the portion of the timing graph that are within each pb-type, leaving a set of of external nodes unconnected until add_net_to_timing_graph?